gitbutler-app/src/bin.rs

```diff
@@ -226,32 +188,10 @@ fn main() {
                 .on_menu_event(|event|menu::handle_event(&event))
                 .build(tauri_context)
                 .expect("Failed to build tauri app")
-                .run(|app_handle, event| match event {
-                    tauri::RunEvent::WindowEvent {
-                        event: tauri::WindowEvent::Focused(is_focused),
-                        ..
-                    } => {
-                        if is_focused {
-                            set_toggle_menu_hide(app_handle)
-                                .expect("Failed to set toggle menu hide");
-                        } else {
-                            set_toggle_menu_show(app_handle)
-                                .expect("Failed to set toggle menu show");
-                        }
-                    }
-                    tauri::RunEvent::ExitRequested { api, .. } => {
-                        hide_window(app_handle).expect("Failed to hide window");
-                        api.prevent_exit();
-                    }
```

Comment:
@krlvi i think this would virtually disable background mode which is a core mac experience. meaning when you click `close`, the app will not just be hidden, but instead stop completely.

Reply:
the `minimize` button is for hiding, `close` is for closing

Reply:
> > @krlvi i think this would virtually disable background mode which is a core mac experience. meaning when you click `close`, the app will not just be hidden, but instead stop completely.
> 
> the `minimize` button is for hiding, `close` is for closing

not on mac though. close on mac closes the window, not the process. the process remains in the dock

Reply:
you are right... without this the red icon kills the process (not the case for VSCode). gonna re-add it

Reply:
@krlvi tauri can't keep the app process running without the tray menu?